components/camel-bindy/src/main/java/org/apache/camel/dataformat/bindy/BindyFixedLengthFactory.java

```diff
@@ -561,6 +561,9 @@ private void initFixedLengthRecordParameters() {
                 // Get carriage return parameter
                 crlf = record.crlf();
                 LOG.debug("Carriage return defined for the CSV: {}", crlf);
+                
+                eol = record.eol();
+                LOG.debug("EOL(end-of-line) defined for the CSV: {}", crlf);
```

Comment:
The debug is using the wrong parameter, it should be eol

Reply:
updated..